templates/galaxy/config/job_conf.xml.j2

```diff
@@ -1,101 +1,139 @@
-		<tool id="interactive_tool_openrefine" destination="interactive_local" />
-		<tool id="interactive_tool_panoply" destination="interactive_local" />
-		<tool id="interactive_tool_paraview" destination="interactive_local" />
-		<tool id="interactive_tool_phinch" destination="interactive_local" />
-		<tool id="interactive_tool_pyiron" destination="interactive_local" />
-		<tool id="interactive_tool_radiant" destination="interactive_local" />
-		<tool id="interactive_tool_rstudio" destination="interactive_local" />
-		<tool id="interactive_tool_simtext_app" destination="interactive_local" />
-		<tool id="interactive_tool_vcf_iobio" destination="interactive_local" />
-		<tool id="interactive_tool_wallace" destination="interactive_local" />
-		<tool id="interactive_tool_wilson" destination="interactive_local" />
+		{# HTcondor #}
+		<destinations default="condor_32">
+			<destination id="local_destination" runner="local_plugin"/>
+			<destination id="local_condor" runner="condor" />
+			<destination id="condor_8" runner="condor">
+				<param id="request_cpus">2</param>
+				<param id="request_memory">8GB</param>
+			</destination>
+			<destination id="condor_16" runner="condor">
+				<param id="request_cpus">4</param>
+				<param id="request_memory">16GB</param>
+			</destination>
+			<destination id="condor_16-fastqc" runner="condor">
+				<param id="request_cpus">2</param>
+				<param id="request_memory">16GB</param>
+				<env id="_JAVA_OPTIONS">-XX:MaxPermSize=2G -Xmx14G -Xms1G</env>
+			</destination>
+			<destination id="condor-shovill" runner="condor">
+				<param id="request_cpus">4</param>
+				<param id="request_memory">40GB</param>
+				<env id="SHOVILL_RAM">40</env>
+			</destination>
+			<destination id="condor_32" runner="condor">
+				<param id="request_cpus">8</param>
+				<param id="request_memory">32GB</param>
+			</destination>
+			<destination id="condor_32-20" runner="condor">
+				<param id="request_cpus">20</param>
+				<param id="request_memory">32GB</param>
+				<param id="requirements">Machine=="galaxy.bioinformatics-atgm.nl"</param>
+			</destination>
+			<destination id="condor_64" runner="condor">
+				<param id="request_cpus">8</param>
+				<param id="request_memory">64GB</param>
+			</destination>
+
+			<destination id="interactive_local" runner="pulsar_embedded">
+				<param id="docker_enabled">true</param>
+				<param id="docker_volumes">$defaults</param>
+				<param id="docker_sudo">false</param>
+				<param id="docker_net">bridge</param>
+				<param id="docker_auto_rm">true</param>
+				<param id="docker_set_user"></param>
+				<param id="require_container">true</param>
+				<param id="request_cpus">1</param>
+				<param id="request_memory">2GB</param>
+				<param id="requirements">Machine=="galaxy.atgm.avans.nl"</param>
+				<param id="container_monitor_result">callback</param>
+			</destination>
+			<destination id="pulsar" runner="pulsar_runner" >
```

Comment:
same here, need 3x, one per runner/plugin.

Reply:
yeah the `runner` name should be customised per destination, so three destinations each to a different runner, use the same logic as your for loop